tools/llm_bench/task/visual_language_generation.py

```diff
@@ -37,7 +37,11 @@ def run_visual_language_generation_optimum(
     prompts = []
     inputs = [inputs] if not isinstance(inputs, (list, tuple)) else inputs
     for input_data in inputs:
-        if input_data.get("media", None):
+        if input_data.get("video", None):
+            entry = Path(input_data["video"])
+            ordered_frames = pu.split_video_into_frames(entry, required_frames)
+            images.extend(ordered_frames)
+        elif input_data.get("media", None):
```

Comment:
`"media"` is images input? if yes, when user inputs video and images in the same time, how to process? because you are using `if else`.

BTW, actually, we also support to input multiple videos, this case should maybe considered.